### PR TITLE
Screen

### PR DESCRIPTION
**Pull request by AbdullahDab01**
Adding screens classes

### CODE REVIEW

File: core/src/View/ScreenEnum.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package View;
+
+public enum ScreenEnum {
+    MENU, SETTINGS , GAME_OVER, GAME
```
Comment (rojtekin) [inline core/src/View/ScreenEnum.java:4]: Start menu instead of menu?

**Review by rojtekin — APPROVED** [review]
LGTM

File: core/src/View/ScreenManger.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package View;
+
+import com.badlogic.gdx.Game;
+
+
+public class ScreenManger  {
```
Comment (pkasinski) [inline core/src/View/ScreenManger.java:6]: Typo in the class name.

**Review by pkasinski — APPROVED** [review]
LGTM

**Pull request merged**